src/org/freedesktop/gstreamer/lowlevel/GType.java

```diff
@@ -98,9 +95,9 @@ public static GType valueOf(long value) {
     public static GType valueOf(String typeName) {
     	GType result = gTypeByNames.get(typeName);
     	if (result == null) {
-    		result = gst.g_type_from_name(typeName);
+    		result = GObjectAPI.GOBJECT_API.g_type_from_name(typeName);
     		if (result.equals(INVALID)) {
-    			// no type has been registered yet
+    			throw new IllegalStateException("No GType registered for this name");
```

Comment:
Ok - replace null in my question with INVALID. What I'm pointing out is, that the behaviour changed - if that is inteded it is ok, but then this should go into the commit message, as the message says something different.